modules/ardrivo/src/Wire.cxx

```diff
@@ -3,78 +3,66 @@
 #include "Arduino.h"
 #include "BoardDataDef.hxx"
 
-//Do diffrently??
-bool transmitting = false;
-uint8_t transmissionAdress;
-uint8_t transmitted;
+//What sets bus_id?? where does it update?
```

Comment:
You can know your bus id by looking up in the `board_info` which bus ĥas the same pins you have